README.md

```diff
@@ -581,6 +651,70 @@ With:
 -DENABLE_CURAND_BACKEND=True   \
 ```
 
+### Building with hipSYCL
+HipSYCL currently supports the BLAS domain with the following backends:
+* NETLIB
+* MKLCPU
+* CUBLAS
+* ROCBLAS
+
+From which the ROCBLAS backend is only available when compiling with hipSYCL
+
+#### Building oneMKL to target AMD GPU with hipSYCL
+
+With the AMD rocBLAS backend:
+
+- On Linux*
+
+```bash
+# Inside <path to onemkl>
+mkdir build && cd build
+cmake .. -DENABLE_CUBLAS_BACKEND=False                     \
+         -DENABLE_MKLCPU_BACKEND=False/True                \   
+         -DENABLE_NETLIB_BACKEND=False/True                \   # hipSYCL supports both NETLIB and MKLCPU backends
+         -DENABLE_MKLGPU_BACKEND=False                     \   # disable Intel MKL GPU backend
+         -DENABLE_ROCBLAS_BACKEND=True                     \
+         -DTARGET_DOMAINS=blas                             \   # hipSYCL only supports the BLAS domain
+         -DHIPSYCL_TARGETS=omp\;hip:gfx906                 \   # Specify the targetted device architectures 
+         -DONEMKL_SYCL_IMPLEMENTATION=hipSYCL              \   # Use the hipSYCL cmake integration
+         [-DREF_BLAS_ROOT=<reference_blas_install_prefix>] \   # required only for testing
+cmake --build .
+ctest
+cmake --install . --prefix <path_to_install_dir>
+```
+
+**AMD GPU device architectures**  
+
+The device architecture can be retrieved via the `rocminfo` tool. The architecture will be displayed in the `Name:` row.
+
+A few often used architectures are listed below:
+| architecture | AMD GPU name |
+| ----         | ----         |
+| gfx906       | Radeon Instinct MI50/60 <br> Radeon (Pro) VII |
+| gfx908       | Radeon Instinct MI 100 |
+| gfx900       | Radeon Instinct MI 25 <br> Radeon RX Vega 64/56 |
+
+#### Building oneMKL to target CUDA GPU with hipSYCL  
```

Comment:
We have not tested this functionality yet (and haven't discussed the coverage). Can we submit this as a separate PR?